pkg/cache/cache.go

```diff
@@ -41,8 +41,23 @@ import (
 var (
 	log               = logf.RuntimeLog.WithName("object-cache")
 	defaultSyncPeriod = 10 * time.Hour
+
+	// BlockUntilSynced determines whether a get request for an informer should block
+	// until the informer's cache has synced.
+	BlockUntilSynced = InformerGetOption(internal.BlockUntilSynced)
```

Comment:
Can we make this a `func BlockUntilSynced() internal.InformerGetOption { return internal.BlockUntilSynced(true) }`

to
* Avoid the package-level var
* The casting back-and-forth shebang?

Reply:
Done, though preserving the boolean arg (since the main concern is to be able to turn off blocking)